httpmw/limiter_v1.go

```diff
@@ -0,0 +1,51 @@
+package httpmw
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+)
+
+// NewRedisLimiter creates redisLimiter.
+func NewRedisLimiter(cmdable redis.Cmdable, limit int, expire time.Duration) *RedisLimiter {
+	return &RedisLimiter{
+		cmdable: cmdable,
+		expire:  expire,
+		limit:   limit,
+	}
+}
+
+type RedisLimiter struct {
+	limit   int
+	expire  time.Duration
+	cmdable redis.Cmdable
+}
+
+// BuildRedisKey returns built key for redis storage.
+func (rl *RedisLimiter) BuildRedisKey(identifier string, entity string) string {
+	return fmt.Sprintf("%s:%s:count", entity, identifier)
+}
+
+// Allow checks identifier is alowed to continue depending on limit.
```

Comment:
I would slightly change this to something like `// Allow checks if this particular identifier can continue performing the operation.`.